src/rpc/trans/RpcSocket.h

```diff
@@ -67,11 +67,9 @@ class RpcSocket : public Object
 	DataPackage recvPacket;
 	bool isWaitingData;
 	String idOfWaitData;
-	SyncLock waitLock;
+	ConditionLock sendLock;
```

Comment:
change into ConditionLock due to changes in the blib that SyncLock no longer provide the method wait() with a mutex.